lib/appcanary/assert.rb

```diff
@@ -0,0 +1,51 @@
+require "json"
+
+module Appcanary
+  class << self
+    def frequencies(arr)
```

Comment:
I would rename this, "count_frequencies" or "frequency_count" :)

so we end up with 

```ruby
count_frequencies(response.map(etc))
```